lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -98,7 +98,11 @@ const defaultSizes = {
   },
 };
 
-const saveSizes = sizes => {
+const saveSizes = (sizes, isMobileDevice) => {
+  if (isMobileDevice) {
```

Comment:
Please add some inline comment explaining why do we need to skip saving here

Reply:
I see two ways of refactoring this part:

1. 

Extract the `saveSizes` and `getSavedSizes` to a separate file and do not change their functionality.
Change only the Layout component to be coupled to the `isMobileDevice`. 

2. 

As an alternative, you can store mobile sizes in a sperate entry of the storage.

Reply:
@igor-dv I like the second option

Reply:
Sorry, I don't really get the point of different entry names. Different devices cannot share the same `localStorage` AFAIK

Reply:
@Hypnosphi  initially I avoided saving dimensions on mobile as the resizing worked not perfectly when I firstly tried it (I did not face issues that you have found later, but still resizing seemed to be a hard task to perform. So I wanted to avoid situations when you somehow resized the pane and did not like the results but the new unwanted sizes are saved in localStorage, so you cannot reset the layout by refreshing the page.

Maybe I misunderstood Igor's comment above about separation of saved dimensions, if it is not needed, but we still want to save sizes on mobile – let's just drop the changes and leave this part as it is in master right now?

Reply:
I talked more about the browser emulators. I think in this case the storage is shared.